packages/tailwind/README.md

```diff
@@ -3,3 +3,159 @@
+
+```
+const { Preset } = require("@kaizen/tailwind")
+
+module.exports = {
+  presets: [Preset],
+  theme: {
+    extend: {
+      colors: {
+        myCoolNewColor: "#ffffff",
+        purple-100: "lime"
+      }
+    }
+  }
+}
+```
+
+Here, the Kaizen preset overwrites the default Tailwind preset, and the `extend` field adds in some new values.  
+Be careful though! While adding in a _new_ field to `colors` won't overwrite any existing ones, passing in an existing field _will_. In this example, `purple-100` has unfortunately been overwritten.
+
+`<p className="text-sky">...` ❌ Default TW config overwritten by `Preset`
+
+`<p className="text-myCoolNewColor">` ✅ New color added
+
+`<p className="text-purple-100">` ️❗️ Value from `Preset` overwritten to "Lime"
+
+`<p className="text-purple-200">` ✅ Value from `Preset` not overwritten
+</details>
+
+<br/>
+
+---
+
+## Suggested Editor Config
+
+The assumption is that you will be using VSCode as your main IDE. Your configuration may differ if you are using alternative like Webstorm, Atom, etc, but there is relatively decent support in their plugin ecosystems
+
+### Tailwindcss VSCode Intellisense plugin
+
+Basic config for the [VSCode plugin](https://marketplace.visualstudio.com/items?itemName=bradlc.vscode-tailwindcss) for Tailwind intellisense lives in your projects `settings.json`.
+
+To get intellisense use:
+
+
+```
+  "tailwindCSS.classAttributes": [
+    "class",
+    "className",
+    "ngClass",
+    "classNameOverride"
+  ]
+```
+
+These identify where tailwind will provide intellisense in your editor.
+
+---
+
+## Prerequisite knowledge
+
+This package is essentially a custom Tailwind preset. This package does not change how Tailwind works; it only changes the  compiled classes.  
+
+For example,
+Tailwind allows us to use atomic classes with psuedo selectors as inline styles, like so:
+```
+<p className="hover:w-1">...
+```
+`w-1` in this instance would compile to a css rule that sets `width: 0.25rem` to the element, if using Tailwind defaults.  
+What our package changes are the presets that can be assigned. After using our preset, the above `w-1` class will not work. Instead, classes that utilize Kaizen's design tokens will be available through our preset.  
+The following example sets a `width` of `2px`, as per preset.
+```
+<p className="hover:w-2">...
+```
+
+Note that the functionality of core concepts like psuedo-selectors and arbitrary values is unchanged.
+
+For this reason, refer to the [Tailwind core concepts docs](https://tailwindcss.com/docs/utility-first) for getting up to speed on how Tailwind _works_, but refer to our [storybook](https://cultureamp.design/storybook/) as a reference guide for _classnames_ made available through our preset.
+
+As an example, if I wanted to know which font-family values are available as Tailwind classes, I would look under Tailwind > Typography > Font Family (or just search "Font Family").
```

Comment:
tbh I was a bit lost where this example was telling me to look this up, might be worth adding `... in our Storybook` to the end of it?

Reply:
Great suggestion, added this in